### ISSUE TITLE
No Name and Role is defined for Icons present in Project screen: A11y_.NET 9 template_Projects_Name Role Value.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26912]
### GitHub Tags:
#A11yTCS; #A11ySev1; #Android; #WCAG4.1.2; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #Device; #A11yMAS; #Name Role Value; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback version: 15.1

### Repro Steps:

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till project section and double tap on any project, Project screen will open,
4. Swipe till category section and choose a category from the list of category.
5. Swipe till tags section and choose a tag from the list of tags or We can select the task from the existing task list.\
6. Swipe till new task button in the project screen and double tap on it.
7. New Project screen gets opened.
8. Swipe till Icon sections and Observe that No Name and Role is defined for Icons present in Project screen.

### Actual Result:
No Name and Role is defined for Icons present in Project screen.

### Expected Result:
Name and Role should be defined for Icons present in Project screen.

### User Impact:
Users who relay on screen reader wont get proper information of those icons.

### Attachment:
https://github.com/user-attachments/assets/66ad43c9-ec90-4a91-b585-9f80807ef708

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by jsuarezruiz** [comment]
Already fixed https://github.com/dotnet/maui/blob/1807a6c10fb5ebeb14c0f24883acb3cbddd97a52/src/Templates/src/templates/maui-mobile/Pages/ProjectDetailPage.xaml#L126

**Comment by keerthiduvvuri** [comment]
Verified in latest Environment, Issue is Fixed. Hence closing the bug.
Added attachment for reference.
#Regressed:03-06-25; #Closed; #A11yFixed;

https://github.com/user-attachments/assets/ef15d764-340a-4ede-a0a9-4a960d31e442
